File: c/12d.c
#include <stdio.h>
#include <string.h>

int main() {
    // imports
    char name[100];
    printf("Enter your name: ");
    scanf("%s", name);

    // define vars
    char line[] = "I must complete my homework on time or the teacher will be cross";

    while (1) {
        for (int i = 0; i < 101; i++) {
            printf("%s\n", line);
        }

        printf("Lines by %s\n", name);

        char userAns[3];
        printf("Again? (yes/no) ");
        scanf("%s", userAns);

        if (strcmp(userAns, "yes") != 0) {
            break;
        }
    }

    return 0;
}

File: python/12d.py
##12d
line = "I must complete my homework on time or the teacher will be cross"
name = str(input("Enter your name > "))
again = 1
while again == 1:
    for i in range (1,101):
        print(line)
    print("Lines by "+name)
    again = int(input("Again? type 1."))